samples/rawvm/k8services.yaml.in

```diff
@@ -17,6 +17,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: raw-svc-deployment
+  labels:
+    app: echosrv
```

Comment:
mismatch on name and app

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416